### PR TITLE
[Release-3_4][Server] Add TextSymbolizer to SLD provided by WMS GetStyles request

### PR DESCRIPTION
**Pull request by rldhont**
## Description
Since QGIS 3.0, it possible to writes the SE 1.1 TextSymbolizer element based on the current layer labeling settings.

Backport of  #32340

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `Fixes #11111` at the bottom of the commit message
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit
- [x] I have evaluated whether it is appropriate for this PR to be backported, backport requests are left as label or comment

### CODE REVIEW

**Comment by rldhont** [comment]
@sbrunner, @elpaso or @pblottiere I have added the test. Can you take a look at this PR ? thanks

File: src/server/services/wms/qgswmsgetstyles.cpp [hunk 1]
```diff
@@ -180,11 +181,37 @@ namespace QgsWms
...
+              QDomElement featureTypeStyleElem = myDocument.createElement( QStringLiteral( "se:FeatureTypeStyle" ) );
+              userStyleElem.appendChild( featureTypeStyleElem );
+
+              vlayer->renderer()->toSld( myDocument, featureTypeStyleElem, props );
+              if ( vlayer->labelsEnabled() )
+              {
+                vlayer->labeling()->toSld( featureTypeStyleElem, props );
+              }
+
+              /*QDomElement styleElem = vlayer->renderer()->writeSld( myDocument, styleName );
```
Comment (elpaso) [inline src/server/services/wms/qgswmsgetstyles.cpp]: can you please remove the comment?

**Review by elpaso — APPROVED** [review]

File: src/server/services/wms/qgswmsgetstyles.cpp (hunk 1 above)
Comment (rldhont) [reply src/server/services/wms/qgswmsgetstyles.cpp]: Done thanks.

**Comment by rldhont** [comment]
Thanks @elpaso

**Pull request merged**